main.go

```diff
@@ -21,6 +21,7 @@ import (
 	"os"
 
 	"github.com/ianschenck/envflag"
+	"go.uber.org/automaxprocs/maxprocs"
 	"github.com/linode/linodego"
```

Comment:
```suggestion
	"github.com/linode/linodego"
	"go.uber.org/automaxprocs/maxprocs"
```
sorry, one more tweak since the groups should be in alphabetical order.

Reply:
Sure!